build.psm1

```diff
@@ -621,6 +630,7 @@ function PlaceUnixBinaries
 
     $RuntimePathLinux = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-x64/native') -Force
     $RuntimePathLinuxARM = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-arm/native') -Force
+    $RuntimePathLinuxARM64 = New-Item -ItemType Directory -Path (Join-Path $PackageRoot -ChildPath 'runtimes/linux-arm64/native') -Force
```

Comment:
Add a parameter to the function for passing in `$BinFolderLinuxARM64`. Then use `$RuntimePathLinuxARM64` like in line 640 for `Copy-Item`

Reply:
Also, `Start-BuildPowerShellNativePackage` function needs a parameter for linux-arm64.